core/data/src/main/java/uk/gov/gchq/gaffer/data/element/Element.java

```diff
@@ -42,7 +42,7 @@
  * <p>
  * Equals has been overridden to check groups are equal. NOTE - it does not compare property values.
  */
-@JsonTypeInfo(use = Id.CLASS, include = As.EXISTING_PROPERTY, property = "class")
+@JsonTypeInfo(use = Id.CLASS, include = As.PROPERTY, property = "class")
```

Comment:
I think this needs to stay as EXISTING_PROPERTY - there is a getter and setter for "class" at the bottom of this class. If you set it to just PROPERTY you might end up with duplicate "class" fields in the json.